backend/graphql/resolvers/testResolvers.ts

```diff
@@ -171,6 +178,12 @@ const testResolvers = {
       });
       return updatedTest;
     },
+    uploadTestImage: async (
+      _req: undefined,
+      { file }: { file: Promise<FileUpload> },
+    ): Promise<ImageMetadata> => {
+      return imageUploadService.uploadImage(file);
```

Comment:
extracted out the image service as standalone bc i forgot i need to use it like this